opencog/nlp/relex2logic/rules/definite.scm

```diff
@@ -18,24 +19,23 @@
                 (VariableNode "$noun")
                 (VariableNode "$a-parse")
             )
-	    (InheritanceLink
-   		(VariableNode "$noun")
-   		(DefinedLinguisticConceptNode "definite")
-	    )
+            (InheritanceLink
+                (VariableNode "$noun")
+                (DefinedLinguisticConceptNode "definite")
+            )
         )
-       (ListLink
-        (ExecutionOutputLink
-       	   (GroundedSchemaNode "scm: pre-definite-rule")
-       	      (ListLink
-       	         (VariableNode "$noun")
+        (ListLink
+            (ExecutionOutputLink
+                (GroundedSchemaNode "scm: pre-definite-rule")
+                (ListLink (VariableNode "$noun"))
             )
         )
-      )
     )
 )
 
-; This is function is not needed. It is added so as not to break the existing
-; r2l pipeline.
+; This is function is not needed. It is added so as not to break the
+; existing r2l pipeline.  Huh ??? How can it not be needed?  It is
```

Comment:
Since we have moved to ure based r2l implementation, there is no need to keep backward compatability, and rules that are added should update the helper functions to accept nodes and not names of the nodes. Then we can avoid the `pre-whatever-helper-function-name` indirections.